wiki/plugins/links/mdx/urlize.py

```diff
@@ -95,6 +95,8 @@ def extendMarkdown(self, md, md_globals):
         md.inlinePatterns['autolink'] = UrlizePattern(URLIZE_RE, md)
 
 def makeExtension(configs=None):
+    if configs is None:
+        configs = {}
```

Comment:
(ignore if this was an upstream change being replayed)

Why not just change the method signature to `def makeExtension(configs={}):`?